src/morphology.cpp

```diff
@@ -50,7 +49,6 @@ Morphology::Morphology(const std::string& source, unsigned int options)
     : Morphology(loadURI(source, options), options) {}
 
 Morphology::Morphology(mut::Morphology morphology) {
-    morphology.sanitize();
```

Comment:
would it make sense to rename `sanitize` to 'remove_unifurcations` or something similar? or does it do more?

Reply:
Done